### PR TITLE
fix(Menu): remove vertical menu background color on focus in dark theme

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
In vertical menu, there's a background color on focus in dark theme:
Screenshot 2022-04-29 at 10 34 57

There should not be background color change on focus This is the behavior after fix:
Screenshot 2022-04-29 at 10 34 28

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2a852c82989e92d10a979979c16be97c30a51027'>2a852c82989e92d10a979979c16be97c30a51027</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
AttachmentSlotsPerf.default | [972](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AttachmentSlotsPerf.default.html) | [883](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.1:1
AvatarMinimalPerf.default | [171](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AvatarMinimalPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.1:1
ButtonSlotsPerf.default | [457](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonSlotsPerf.default.html) | [429](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.07:1
CardMinimalPerf.default | [462](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CardMinimalPerf.default.html) | [437](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.06:1
LayoutMinimalPerf.default | [295](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LayoutMinimalPerf.default.html) | [278](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.06:1
PortalMinimalPerf.default | [145](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/PortalMinimalPerf.default.html) | [137](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.06:1
IconMinimalPerf.default | [497](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/IconMinimalPerf.default.html) | [471](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.06:1
DatepickerMinimalPerf.default | [4939](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4748](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.04:1
LabelMinimalPerf.default | [327](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LabelMinimalPerf.default.html) | [314](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.04:1
TableManyItemsPerf.default | [1631](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TableManyItemsPerf.default.html) | [1575](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.04:1
AnimationMinimalPerf.default | [463](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AnimationMinimalPerf.default.html) | [449](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.03:1
DividerMinimalPerf.default | [300](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DividerMinimalPerf.default.html) | [292](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.03:1
FlexMinimalPerf.default | [240](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/FlexMinimalPerf.default.html) | [232](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.03:1
HeaderMinimalPerf.default | [300](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/HeaderMinimalPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.03:1
InputMinimalPerf.default | [1105](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/InputMinimalPerf.default.html) | [1074](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.03:1
CarouselMinimalPerf.default | [382](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CarouselMinimalPerf.default.html) | [373](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.02:1
FormMinimalPerf.default | [341](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/FormMinimalPerf.default.html) | [333](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.02:1
LoaderMinimalPerf.default | [578](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LoaderMinimalPerf.default.html) | [565](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.02:1
PopupMinimalPerf.default | [531](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/PopupMinimalPerf.default.html) | [521](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
ProviderMinimalPerf.default | [335](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ProviderMinimalPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.02:1
RadioGroupMinimalPerf.default | [373](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [365](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.02:1
ReactionMinimalPerf.default | [314](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ReactionMinimalPerf.default.html) | [309](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.02:1
SkeletonMinimalPerf.default | [287](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SkeletonMinimalPerf.default.html) | [281](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.02:1
SliderMinimalPerf.default | [1397](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SliderMinimalPerf.default.html) | [1367](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.02:1
TableMinimalPerf.default | [339](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TableMinimalPerf.default.html) | [332](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.02:1
ButtonOverridesMissPerf.default | [1250](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1242](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [617](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatMinimalPerf.default.html) | [611](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
CheckboxMinimalPerf.default | [2262](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2244](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [569](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DropdownManyItemsPerf.default.html) | [565](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [1000](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [991](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
MenuMinimalPerf.default | [721](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/MenuMinimalPerf.default.html) | [715](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.01:1
ProviderMergeThemesPerf.default | [1059](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1048](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.01:1
StatusMinimalPerf.default | [566](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/StatusMinimalPerf.default.html) | [560](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.01:1
TextMinimalPerf.default | [285](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TextMinimalPerf.default.html) | [282](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.01:1
TextAreaMinimalPerf.default | [396](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TextAreaMinimalPerf.default.html) | [393](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.01:1
TooltipMinimalPerf.default | [909](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TooltipMinimalPerf.default.html) | [897](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.01:1
BoxMinimalPerf.default | [275](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/BoxMinimalPerf.default.html) | [276](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1:1
ChatDuplicateMessagesPerf.default | [239](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [238](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1:1
DropdownMinimalPerf.default | [2571](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DropdownMinimalPerf.default.html) | [2560](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [3439](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/EmbedMinimalPerf.default.html) | [3446](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
GridMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/GridMinimalPerf.default.html) | [276](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/GridMinimalPerf.default_base.html) | 1:1
HeaderSlotsPerf.default | [626](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/HeaderSlotsPerf.default.html) | [625](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1:1
ListCommonPerf.default | [515](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListCommonPerf.default.html) | [515](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListCommonPerf.default_base.html) | 1:1
ListNestedPerf.default | [460](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListNestedPerf.default.html) | [459](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListNestedPerf.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [3666](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3675](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [2266](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CustomToolbarPrototype.default.html) | [2260](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
ListWith60ListItems.default | [529](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListWith60ListItems.default.html) | [536](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.99:1
MenuButtonMinimalPerf.default | [1422](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1430](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 0.99:1
ToolbarMinimalPerf.default | [789](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ToolbarMinimalPerf.default.html) | [798](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.99:1
TreeMinimalPerf.default | [687](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TreeMinimalPerf.default.html) | [691](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TreeMinimalPerf.default_base.html) | 0.99:1
RosterPerf.default | [911](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RosterPerf.default.html) | [930](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RosterPerf.default_base.html) | 0.98:1
AccordionMinimalPerf.default | [114](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AccordionMinimalPerf.default.html) | [118](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.97:1
AlertMinimalPerf.default | [221](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AlertMinimalPerf.default.html) | [227](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.97:1
AttachmentMinimalPerf.default | [119](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AttachmentMinimalPerf.default.html) | [123](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 0.97:1
ImageMinimalPerf.default | [299](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ImageMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.97:1
ListMinimalPerf.default | [417](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListMinimalPerf.default.html) | [431](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ListMinimalPerf.default_base.html) | 0.97:1
VideoMinimalPerf.default | [534](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/VideoMinimalPerf.default.html) | [550](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/VideoMinimalPerf.default_base.html) | 0.97:1
ButtonMinimalPerf.default | [127](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonMinimalPerf.default.html) | [132](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.96:1
DialogMinimalPerf.default | [608](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DialogMinimalPerf.default.html) | [632](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/DialogMinimalPerf.default_base.html) | 0.96:1
RefMinimalPerf.default | [191](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RefMinimalPerf.default.html) | [198](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.96:1
TreeWith60ListItems.default | [132](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TreeWith60ListItems.default.html) | [138](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/TreeWith60ListItems.default_base.html) | 0.96:1
SegmentMinimalPerf.default | [270](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SegmentMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.95:1
ChatWithPopoverPerf.default | [293](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatWithPopoverPerf.default.html) | [313](https://fluentuipr.z22.web.core.windows.net/pull/22707/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.94:1

</details><div id="perfComment9424"/>

File: packages/fluentui/react-northstar/src/themes/teams-dark/components/Menu/menuVariables.ts [hunk 1]
```diff
@@ -33,7 +33,6 @@ export const menuVariables = (siteVars: any): Partial<MenuVariables> => ({
   activeUnderlinedBorderBottomColor: siteVars.colors.white,
   activeUnderlinedPrimaryColor: siteVars.colors.brand[400],
 
-  verticalBackgroundColorFocus: siteVars.colors.grey[550],
```
Comment (miroslavstastny) [inline packages/fluentui/react-northstar/src/themes/teams-dark/components/Menu/menuVariables.ts:36]: This was the only place where the variable was set. Does it make sense to keep the variable or should we remove it completely?

Comment (YuanboXue-Amber) [reply packages/fluentui/react-northstar/src/themes/teams-dark/components/Menu/menuVariables.ts:36]: good point, I think it should be removed

**Comment by ling1726** [comment]
`There should not be a background color change on focus` should this be the case for HC too ?

image

**Comment by YuanboXue-Amber** [comment]
> `There should not be a background color change on focus` should this be the case for HC too ?
> 
> image

yeah it should. This e2e should not change. I'll fix it

**Pull request merged**